extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/QuarkusErrorHandler.java

```diff
@@ -200,4 +237,45 @@ static String escapeJsonString(final String text) {
         return sb.toString();
     }
 
+    private static final class ContentTypes {
+
+        private ContentTypes() {
+        }
+
+        private static final String APPLICATION_JSON = "application/json";
+        private static final String TEXT_JSON = "text/json";
+        private static final String TEXT_HTML = "text/html";
+        private static final String APPLICATION_XML = "application/xml";
+        private static final String TEXT_XML = "text/xml";
+        private static final String APPLICATION_XHTML = "application/xhtml+xml";
+        private static final Set<String> SUPPORTED_SET = Set.of(APPLICATION_JSON, TEXT_JSON, TEXT_HTML, APPLICATION_XML,
+                TEXT_XML, APPLICATION_XHTML);
+
+        static String pickFirstSupportedAndAcceptedContentType(HttpServerRequest request) {
```

Comment:
I'm wondering if Vert.x does not have this code somewhere.

Reply:
Should we reuse: https://github.com/vert-x3/vertx-web/blob/36bf548a7362336a9838e8260c40b543fc032cc6/vertx-web/src/main/java/io/vertx/ext/web/impl/HeaderParser.java ?

Reply:
Thanks! I was looking for something like that, but didn't find it. I'll change the code to use it.